bom/runtime/pom.xml

```diff
@@ -1551,7 +1551,29 @@
             </dependency>
             <dependency>
                 <groupId>io.smallrye</groupId>
-                <artifactId>smallrye-open-api</artifactId>
+                <artifactId>smallrye-open-api-core</artifactId>
+                <version>${smallrye-open-api.version}</version>
```

Comment:
Are these not excluded from the spec? Or excluded from SmallRye dependency import?

Reply:
Let me check this...

Reply:
This is not being excluded in SmallRye or the Spec, so for now it needs to stay here ? I'll add it to the next version of SR.